validated/tables.py

```diff
@@ -1,66 +1,7 @@
-from dash_ag_grid import AgGrid
-
-
-def create_daily_table(data: dict) -> AgGrid:
```

Comment:
Why these two need to be removed from there? Is it a dash thing?

Reply:
Yeah, a problem with the callback not recognizing the object. It's ok to build objects with functions, but for some reason those functions need to be in the same file as the callback or it has problems. Ran into similar issues with Gridlington.